lib/splitclient-rb/validators.rb

```diff
@@ -38,6 +41,25 @@ def valid_matcher_arguments(args)
       true
     end
 
+    def valid_flag_sets(method, flag_sets)
+      if flag_sets.nil? || !flag_sets.is_a?(Array)
```

Comment:
you can replace this if for one line: `return nil if flag_sets.nil? || !flag_sets.is_a?(Array)`

Reply:
will do